tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -265,4 +266,46 @@ public void testUnixSocket() throws Exception
             Files.deleteIfExists(sockFile);
         }
     }
+
+    @Test
+    public void testLog4j2ModuleWithSimpleWebAppWithJSP() throws Exception
+    {
+        Path jettyBase = Files.createTempDirectory( "jetty_base");
+        String jettyVersion = System.getProperty("jettyVersion");
+        DistributionTester distribution = DistributionTester.Builder.newInstance()
+            .jettyVersion(jettyVersion)
+            .jettyBase(jettyBase)
+            .mavenLocalRepository(System.getProperty("mavenRepoPath"))
+            .build();
+
+        String[] args1 = {
+            "--create-startd",
+            "--approve-all-licenses",
+            "--add-to-start=resources,server,http,webapp,deploy,jsp,servlet,servlets,logging-log4j2"
```

Comment:
Can you test that the `${jetty.base}/resources/` has the log4j2.xml file?

Reply:
for my curiosity what is the difference with testing that few lines later after the start?

Reply:
@joakime got it fixed